CHANGELOG.md

```diff
@@ -17,8 +17,8 @@
 ### ⚠️ Breaking Changes ⚠️
 - ABI: Implemented a new callback-interface ABI that significantly improves performance on Python and Kotlin.
   - UniFFI users will automatically get the benefits of this without any code changes.
-  - External bindings authors will need to update their bindings code.  See PR #1494 for details.
-- ABI: Changed API checksum handling.  This affects external bindings authors who will need to update their code to work with the new system.  See PR #1469 for details.
+  - External bindings authors will need to update their bindings code.  See [PR #1494](https://github.com/mozilla/uniffi-rs/pull/1494) for details.
```

Comment:
I know you only added the link, but I don't think that PR is particularly useful - nothing actually landed in that PR. Maybe you could add https://github.com/mozilla/uniffi-rs/commit/323a4976992aff207db7946fc1f1cea614838f46 (which is roughly the outcome of the discussion in that PR) to the list below and change this para to just say "please see "Guidance for external bindings below" here?

Reply:
yup, I asked in channel. That commit looks much better to link to indeed.